ui/raidboss/data/triggers/byakko-ex.js

```diff
@@ -6,67 +6,119 @@
     {
       id: 'ByaEx Heavenly Strike',
       regex: / 14:27DA:Byakko starts using Heavenly Strike on (\y{Name})/,
+      regexDe: / 14:27DA:Byakko starts using Himmelszorn on (\y{Name})/,
       alertText: function(data, matches) {
-        if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
-        if (data.role == 'healer')
-          return 'Buster on ' + data.ShortName(matches[1]);
+        if (matches[1] == data.me){
```

Comment:
Sorry, can you add a space before the curly braces here and elsewhere in this file?

Reply:
Sure, no problem ^^